extensions/amazon-sales/release-notes/index.md

```diff
@@ -9,7 +9,7 @@ See the following documentation:
 
 - [Amazon Sales Channel](https://docs.magento.com/m2/ce/user_guide/sales-channels/amazon/amazon-sales-channel.html) for merchant information and instructions
 - [Amazon Sales Channel install]({{site.baseurl}}/extensions/amazon-sales/) for installation and API key information
-- [Amazon Sales Channel Marketplace page](http://marketplace.magento.com/magento-module-amazon.html)
+- Amazon Sales Channel Marketplace page
```

Comment:
Hi @deepakcedcoss 
Please remove the whole line( line no 12).
Because these links are related to document so we can't show only text here without document link.

Reply:
Hi @deepakcedcoss.  Please visit the page again and verify the link doesn't work.  The release notes page that you want to change is located at: https://devdocs.magento.com/extensions/amazon-sales/release-notes/.    I am unable to duplicate your error.  The link works for me.  https://marketplace.magento.com/magento-module-amazon.html

Reply:
Hi @deepakcedcoss. If I don't hear back from you by Monday, July 29, I'll go ahead and close this PR.  Let me know if the link still does not work for you.